src/status_im2/contexts/chat/messages/content/reactions/view.cljs

```diff
@@ -27,5 +26,7 @@
                                                    :emoji-id   emoji-id}]))
             :accessibility-label (str "emoji-reaction-" emoji-id)}]])
        [quo/add-reaction
-        {:on-press #(rf/dispatch [:bottom-sheet/show-sheet
-                                  {:content (fn [] [drawers/reactions chat-id message-id])}])}]])))
+        {:on-press #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals